source/subs_functions/subs_functions.bas

```diff
@@ -2058,6 +2058,19 @@ id.arg = MKL$(ULONGTYPE - ISPOINTER)
 id.hr_syntax = "_SNDCLOSE handle&"
 regid
 
+' a740g: Feature request #28
+clearid
+id.n = qb64prefix$ + "SndNew" ' Name in CaMeL case
+id.Dependency = DEPENDENCY_AUDIO_OUT ' QB64-PE library dependency
+id.subfunc = 1 ' 1 = function, 2 = sub
+id.callname = "func__sndnew" ' C/C++ function name
+id.args = 3 ' number of arguments "passed"
+id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) ' arguments & types

```

Comment:
Did you intend for this to list 4 arguments instead of 3?

Reply:
That's a good catch. I initially wanted 4 args. The 4th one being frequency. But I dropped it. Now I cannot remember why.  
I wanted frames to be mandatory with the rest being optional.
But yeah. This is clearly an issue. I forgot to change `id.arg`.